src/App.tsx

```diff
@@ -1,61 +1,132 @@
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import { useState } from 'react';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+import { User } from './types/User';
+import { Todo } from './types/Todo';
+import { TodoList } from './components/TodoList';
+
+function getUserById(userId: number): User | null {
+  return usersFromServer.find(user => user.id === userId) || null;
+}
+
+export const todos: Todo[] = todosFromServer.map((todo) => ({
+  ...todo,
+  user: getUserById(todo.userId),
+}));
 
 export const App = () => {
+  const [visibleTodos, setVisibleTodos] = useState<Todo[]>(todos);
+  const [title, setTitle] = useState('');
+  const [selectedUser, setSelectedUser] = useState('0');
```

Comment:
you store just id, not a user
```suggestion
  const [selectedUserId, setSelectedUserId] = useState('0');
```